openapi.yaml

```diff
@@ -317,14 +319,15 @@ info:
     }
     ```
     
-    This tells the back-end that the process expects the result (i.e. the return value) from another node to be passed as argument. The `<ProcessNodeIdentifier>` is strictly scoped and can only referenced from within the same process graph, i.e. can not be referenced in child or parent process graphs.
+    This tells the back-end that the process expects the result (i.e. the return value) from another node to be passed as argument.
+    The `<ProcessNodeIdentifier>` is strictly scoped and can only reference nodes from within the same process graph, not child or parent process graphs.
 
     ### Accessing process parameters
 
     **Example:** You want to iterate over an array and calculate the absolute value of each value in the array.
     This can be achieved by executing the `apply` process in openEO and pass a process graph containing a single process `absolute`.
```

Comment:
```suggestion
    This can be achieved by executing the `array_apply` process in openEO and pass a process graph containing a single process `absolute`.
```